<launch>


  <arg name="marker_size" default="15" />
  <arg name="max_new_marker_error" default="0.05" />
  <arg name="max_track_error" default="0.05" />
  <arg name="cam_image_topic" default="/io/internal_camera/head_camera/image_raw" />
  <arg name="cam_info_topic" default="/io/internal_camera/head_camera/camera_info" />
  <arg name="output_frame" default="/head_camera" />

  <node name="ar_track_alvar_sawyer" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" >
      <remap from="/visualization_marker" to="/visualization_marker_sawyer"/>
      <remap from="/ar_pose_marker" to="/ar_pose_marker_sawyer"/>

  </node>






</launch>
